src/components/search.js

```diff
@@ -10,7 +10,7 @@ import Img from 'gatsby-image'
 
 const styles = theme => ({
   margin: {
-    margin: theme.spacing.unit,
+    margin: theme.spacing(1),//new syntax for spacing in mui
```

Comment:
Can you please remove the comment, and just add a hint in the commit if needed, or in the github pull request?